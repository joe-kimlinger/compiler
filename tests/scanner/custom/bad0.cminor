// length
// length = 254 + 2 quotes and null = 257
s1: string = "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234"
