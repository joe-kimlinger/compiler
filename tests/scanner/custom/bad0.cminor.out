COMMENT
COMMENT
IDENTIFIER s1
COLON
STRING
ASSIGN
